std/algebra/twistededwards/point.go

```diff
@@ -150,7 +145,7 @@ func (p *Point) ScalarMul(api frontend.API, p1 *Point, scalar frontend.Variable,
 // DoubleBaseScalarMul computes s1*P1+s2*P2
 // where P1 and P2 are points on a twisted Edwards curve
 // and s1, s2 scalars.
-func (p *Point) DoubleBaseScalarMul(api frontend.API, p1, p2 *Point, s1, s2 frontend.Variable, curve EdCurve) *Point {
+func (p *Point) DoubleBaseScalarMul(api frontend.API, p1, p2 *Point, s1, s2 frontend.Variable, curve *CurveParams) *Point {
```

Comment:
Couldn't this also use GLV?

Reply:
It could but constraint-wise it's worse. It would need a `Lookup4`. The analysis is similar to a 2-bit window vs. a 4-bit window.